operator/src/api.js

```diff
@@ -16,17 +15,17 @@ const {recoverTypedSignature_v4} = require('eth-sig-util');
  */
 
 // TODO: Consider moving to config
-const GET_ENCRYPTION_PUB_KEY_GAS_PRICE = 0.001;
-const GET_ENCRYPTION_PUB_KEY_GAS_LIMIT = 4712388;
-const EXECUTE_DEAL_GAS_PRICE = 0.001;
-const EXECUTE_DEAL_BASE_GAS_UNIT = 6000000;
-const EXECUTE_DEAL_PARTICIPANT_GAS_UNIT = 24000000;
+const GET_ENCRYPTION_PUB_KEY_GAS_PRICE = 1e-8;
+const GET_ENCRYPTION_PUB_KEY_GAS_LIMIT = 0.05e+8;
+const EXECUTE_DEAL_GAS_PRICE = 1e-8;
+const EXECUTE_DEAL_BASE_GAS_UNIT = 0.05e+8;
+const EXECUTE_DEAL_PARTICIPANT_GAS_UNIT = 1e+8;
```

Comment:
Are you using the scientific notation on purpose? If so, why?

Reply:
I optimized the secret contract a week or two ago, if you recall. I've been measuring the gas units consumed, and saw that they were orders of magnitude lower compared to what they must have been when you measured. I then adjusted the gas price so that gas-price * gas-limit was a reasonable amount of ENG, which i decided should be around 0.1-2 ENG per operation (considering the full ENG supply is only 150,000,000). It turns out that the lowest gas price, 1e-8 is the only one that satisfies this constraint, which leads to a conversation i'm going to have with moria today about potentially dividing the gas-units consumed in the secret contract tasks by 1000 or similar, to allow finer grained pricing.